server/events/plan_command_runner.go

```diff
@@ -130,10 +135,6 @@ func (p *PlanCommandRunner) run(ctx *CommandContext, cmd *CommentCommand) {
 	baseRepo := ctx.Pull.BaseRepo
 	pull := ctx.Pull
 
-	if err = p.commitStatusUpdater.UpdateCombined(baseRepo, pull, models.PendingCommitStatus, models.PlanCommand); err != nil {
```

Comment:
This should stay here IMHO, I don't see it as an expensive or noisy operation but it indicates that Atlantis received the request and processing it. If there are no plans to process you can exit and update this status.